### PR TITLE
rm unused  return

### PR DESCRIPTION
**Pull request by NacerKH**
i know is a typo 🤣

### CODE REVIEW

**Comment by HijenHEK** [comment]
Hi @khalifa-dv ,

it's an old code to generate sanctum tokens,  where I used the `Laravel\Sanctum\HasApiTokens`
then I switched to passport OAuth server implementation and just left it there x), thanks for noting that.

I can't accept this pull request since it changed the `composer.lock` file can you please ignore it?

**Comment by NacerKH** [comment]
@HijenHEK 
okay

File: app/Http/Controllers/api/v1/AdminAuthController.php [hunk 1]
```diff
@@ -37,6 +37,13 @@ public function login(Request $request)
 
         $passport_client = Passport::client()->where('name', "admins")->first();
 
+        if (empty($passport_client)){
```
Comment (HijenHEK) [inline app/Http/Controllers/api/v1/AdminAuthController.php]: i think here we should provide a server error to the client 
```php
abort_unless($passport_client, 500, 'authentication server error')
```
reason is that the client would be confused about his credentials

**Review by HijenHEK — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/api/v1/AdminAuthController.php (hunk 1 above)
Comment (NacerKH) [reply app/Http/Controllers/api/v1/AdminAuthController.php]: I agree that I should return abort(500) 'a server error to the client' to make the error specific.

**Review by HijenHEK — APPROVED** [review]

**Pull request merged**